packages/app/src/usePromiseFn.ts

```diff
@@ -62,9 +63,9 @@ export const usePromiseFn = <TFunc extends AsyncFunction>(
           setState({ type: "rejected", error })
       );
       return promise;
-      // eslint-disable-next-line react-hooks/exhaustive-deps
     },
-    [...deps, currentValue]
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [...deps, currentValue, promiseFn]
```

Comment:
IIRC it was intentional/important that `promiseFn` was not included in this, but can't recall why. I should prob write some tests to confirm

Reply:
Sounds good, will undo this!